schemas/schema-schema.ipldsch

```diff
@@ -1,704 +1 @@
-## This representation type encodes in the same way as
-## MapStructRepresentation_Tuple. It is also similar to
-## StructRepresentation_Tuple except it includes the keys in nested lists.
-## A tuple representation for a struct will encode more compact than listpairs.
-##
-type StructRepresentation_ListPairs struct {}
-
-## TypeEnum describes a type which has a known, pre-defined set of possible
-## values. Each of the values must be representable as a string (EnumValue)
-## when using the default "string" representation, or when using an "int"
-## representation, an integer must also be supplied along with the EnumValue.
-##
-## Integer and string values (for int and string representations respectively)
-## are provided in parens in the DSL. Where the string used in serialization is
-## the same as the EnumValue, it may be omitted. For int representation enums,
-## all int values are required.
-##
-type TypeEnum struct {
-	members {EnumValue:Null}
-	representation EnumRepresentation
-}
-
-## EnumValue is a string that has limitations for use as a member of an enum
-## set. The rules for EnumValue are the same as for TypeName but without the
-## convention of an uppercase first character. Capitalization is left up to
-## the user.
-##
-type EnumValue string
-
-## EnumRepresentation describes how an enum type should be mapped onto
-## its IPLD Data Model representation. By default an enum is represented as a
-## string kind but it may also be represented as an int kind.
-##
-type EnumRepresentation union {
-	| EnumRepresentation_String "string"
-	| EnumRepresentation_Int "int"
-} representation keyed
-
-## EnumRepresentation_String describes the way an enum is represented as a
-## string in the data model. By default, the strings used as EnumValue will be
-## used at the serialization. A custom string may be provided (with `Foo ("x")`
-## in the DSL) which will be stored here in the representation block. Missing
-## entries in this map will use the default.
-##
-type EnumRepresentation_String {EnumValue:String}
-
-## EnumRepresentation_Int describes the way an enum is represented as an int
-## in the data model. A mapping of names to ints is required to perform the
-## conversion from int to enum value. In the DSL, int values _must_ be provided
-## for each EnumValue (with `Foo ("100")`, those are stored here.
-##
-type EnumRepresentation_Int {EnumValue:Int}
-
-## TypeCopy describes a special "copy" unit that indicates that a type name
-## should copy the type descriptor of another type. TypeCopy does not redirect a
-## name to another type. Instead, it copies the entire type definition and
-## assigns it to another type.
-##
-## The DSL defines a TypeCopy as `type NewThing = CopiedThing`, where
-## "CopiedThing" refers to a `type` defined elsewhere in a schema and is not
-## one of TypeKind or an inline type descriptor (`{}`, `[]`, `&`).
-##
-type TypeCopy struct {
-	fromType TypeName
-}
+# This document has moved: https://ipld.io/docs/schemas/
```

Comment:
We should really maintain this in code in a canonical place, and i think this is the right place, so that other language implementations can base their schema parsing off of this definition.

The schema-schema is not linked or visible on the ipld.io docs page, so this is not a reasonable deprecation

Reply:
The bug here is that the linking in the new site didn't pick up the schema-schema file in its new home there.  It's a good callout, but we should just fix that there.